Makefile

```diff
@@ -454,6 +455,11 @@ docs/admin/prometheus.md: scripts/metricsdocgen/main.go scripts/metricsdocgen/me
 	cd site
 	yarn run format:write ../docs/admin/prometheus.md
 
+coderd/apidocs/swagger.json: $(shell find scripts/apidocgen -type f | grep -v node_modules) $(wildcard coderd/*.go)
```

Comment:
Suggestion: Omit `node_modules` from `find` scope to prevent walking the excessive tree. You can do this with `-prune`. I'd refer to an example above but unfortunately they're incorrect.

It's not pretty... but it's fast!

```
❯ time find . -not \( \( -path './.git/*' -o -path './build/*' -o -path './vendor/*' -o -path './.coderv2/*' -o -path './site/node_modules/*' -o -path './site/out/*' \) -prune \) -type f -name '*.go' >/dev/null
find . -not \( \( -path './.git/*' -o -path './build/*' -o -path './vendor/*'  0.03s user 0.01s system 98% cpu 0.039 total
```

This one is currently in use in the Makefile, sloow.

```
❯ time find . -not \( -path './.git/*' -o -path './build/*' -o -path './vendor/*' -o -path './.coderv2/*' -o -path './site/node_modules/*' -o -path './site/out/*' \) -type f -name '*.go' >/dev/null
find . -not \( -path './.git/*' -o -path './build/*' -o -path './vendor/*' -o  0.15s user 0.25s system 99% cpu 0.403 total
```

The syntax would be essentially:

```
find scripts/apidocgen -not \( -path './path/to/node_modules/*' -prune \) -type f
```

Needs a second layer of parens only if you're adding more paths because find syntax is a horror show.

Reply:
Fixed